sql/mozfun/glam/histogram_fill_buckets_dirichlet/udf.sql

```diff
@@ -0,0 +1,70 @@
+-- udf_fill_buckets
+CREATE OR REPLACE FUNCTION glam.histogram_fill_buckets_dirichlet(
+  input_map ARRAY<STRUCT<key STRING, value FLOAT64>>,
+  buckets ARRAY<STRING>,
+  total_users INT64
```

Comment:
I noticed there are two different `udf_fill_buckets` right now: one that divides by total users and one that doesn't.  Is it appropriate to have this dividing one be the persistent udf?

Reply:
Yes, this is the variant used to aggregate histograms. I think it should replace `udf_fill_bucket` in scalar aggregates too, for consistency.